### ISSUE TITLE
Comment count

### ISSUE THREAD

**Issue by CarlosTorrecillas** [MurhafSousli/ngx-disqus#62]
Any chance this library could include the relevant code to add the disqus comment count?

Thanks for the great project by the way! Really useful

**Comment by MurhafSousli** [comment]
Isn't there a comment count?

![Screenshot 2020-05-26 at 18 31 12](https://user-images.githubusercontent.com/8130692/82926163-15957200-9f7f-11ea-8617-3c4ceca60564.png)

**Comment by CarlosTorrecillas** [comment]
Yes, sorry @MurhafSousli I perhaps explained myself incorrectly. What I wanted to achieve through that library as it is part of the same stuff is the faq that you could print the comment count next to a blog or article summary like this: https://help.disqus.com/en/articles/1717274-adding-comment-count-links-to-your-home-page

**Comment by MurhafSousli** [comment]
Ah ok, I remember I tried to integrate this in the plugin long time ago. but it didn't work with me, if you find a way to do it you can submit a PR!

**Comment by CarlosTorrecillas** [comment]
cool thanks!
